io/FV3GFS_io.F90

```diff
@@ -31,9 +31,12 @@ module FV3GFS_io_mod
   use constants_mod,      only: grav, rdgas
 !
 !--- GFS physics modules
+#if 1
```

Comment:
I updated this file, please have a look. Thanks!

Reply:
Please do not commit this code yet until it is merged with fv3atm master.
I just found out the GFS_typedef has old code in there such as "hprim", "lgocart" etc.
IPlease verify that it is merged before committing.
Thanks
Moorthi

Reply:
Hi Moorthi, they code is 100% up to date with the latest master.

Can you please clarify where hprim is in GFS_typedefs.F90? I searched the file, the only one used is Sfprop%hprime. The interstitial variable Interstitial%hprime1 has been removed completely. The one definition that was left mistakenly is in lines 1042-1043 (inside an `#ifdef CCPP` block):
```
    ! Temporary to support transition to GFSv16 physics
    logical :: lgocart = .false.
```
but since it is not used I can remove it (or leave the cleanup for another commit if Jun/Dusan do not want to change the code anymore for regression testing):
```
imk-ifu-l257b1:ufs-weather-model_update_from_gmtb-develop_20191016 heinzeller-d$ grep -Rie 'lgocart' FV3/gfsphysics/GFS_layer/GFS_typedefs.* FV3/ccpp
FV3/gfsphysics/GFS_layer/GFS_typedefs.F90:    logical :: lgocart = .false.
FV3/gfsphysics/GFS_layer/GFS_typedefs.meta:[lgocart]
```

Reply:
@junwang-noaa @DusanJovic-NOAA please let me know if you want me to remove lines 1042-1043 (depending on how far your regression tests got). Thanks! And thanks @SMoorthi-emc for spotting these two lines.